.vscode/tasks.json

```diff
@@ -0,0 +1,26 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "shell",
+			"label": "make: build",
+			"command": "make build",
+			"group": {
+                "kind": "build",
```

Comment:
Let's fix the formatting here.

Reply:
Tempting to add an `.editorconfig` to control the tab chars.